### ISSUE TITLE
some things arent importing

### ISSUE THREAD

**Issue by xmcdizzle** [Jed-Giblin/quicksim#1]
heyo, 

installed addon, ran it and copied the JSON code into osthy's sim, everything pasted in just fine, but there were some enchants that did not select when importing. 

those included: shoulder enchants from hodir, tailoring embroidery, wrist enchant, engi gloves, leg spellthread, and nitro boots

**Comment by Jed-Giblin** [comment]
Hey. I’m traveling and on mobile only for a bit so I won’t be able to dig
in, but please upload your export and I’ll take a look when I return.

The addon will export your gear and enchants as is, while the sim only
supports a limited number of selections.

Also sometimes, the enchant names vs effects are different from in game vs
the sim.

**Comment by xmcdizzle** [comment]
yeah of course, no problem. im not one of those who are clueless to the sim, so this is just me pointing out little things so others dont have to. 

{"items":[{"enchant":{"id":3820,"name":"Arcanum of Burning Mysteries","itemId":59970},"name":"Visage Liquification Goggles","slot":"HEAD","id":42553,"gems":[{"id":41285,"name":"Chaotic Skyflare Diamond"},{"id":40051,"name":"Reckless Monarch Topaz"}]},{"id":40680,"gems":[],"name":"Encircling Burnished Gold Chains","slot":"NECK"},{"enchant":{"id":3810,"name":"Greater Inscription of the Storm","itemId":59937},"name":"Valorous Frostfire Shoulderpads","slot":"SHOULDERS","id":40419,"gems":[{"id":40051,"name":"Reckless Monarch Topaz"}]},{"enchant":{"id":3722},"name":"Deathchill Cloak","slot":"BACK","id":41610,"gems":[]},{"enchant":{"id":3832,"spellId":60692,"name":"Enchant Chest - Powerful Stats"},"name":"Valorous Frostfire Robe","slot":"CHEST","id":40418,"gems":[{"id":39998,"name":"Runed Scarlet Ruby"},{"id":40051,"name":"Reckless Monarch Topaz"}]},{"enchant":{"id":2326,"spellId":44635,"name":"Enchant Bracers - Greater Spellpower"},"name":"Cuffs of Winged Levitation","slot":"WRISTS","id":37361,"gems":[]},{"id":40696,"gems":[{"id":40051,"name":"Reckless Monarch Topaz"},{"id":40026,"name":"Purified Twilight Opal"}],"name":"Plush Sash of Guzbah","slot":"WAIST"},{"enchant":{"id":3872},"name":"Valorous Frostfire Leggings","slot":"LEGS","id":40417,"gems":[{"id":39998,"name":"Runed Scarlet Ruby"},{"id":40051,"name":"Reckless Monarch Topaz"}]},{"enchant":{"id":3606},"name":"Sandals of Crimson Fury","slot":"FEET","id":44202,"gems":[{"id":40026,"name":"Purified Twilight Opal"}]},{"enchant":{"id":3604},"name":"Gloves of Token Respect","slot":"HANDS","id":39733,"gems":[]},{"id":40719,"gems":[],"name":"Band of Channeled Magic","slot":"FINGER_1"},{"id":44283,"gems":[],"name":"Signet of Hopeful Light","slot":"FINGER_2"},{"id":40682,"gems":[],"name":"Sundial of the Exiled","slot":"TRINKET_1"},{"id":40255,"gems":[],"name":"Dying Curse","slot":"TRINKET_2"},{"enchant":{"id":3834,"spellId":60714,"name":"Enchant Weapon - Mighty Spellpower"},"name":"The Turning Tide","slot":"MAIN_HAND","id":40396,"gems":[]},{"id":40273,"gems":[],"name":"Surplus Limb","slot":"OFF_HAND"},{"id":39426,"gems":[],"name":"Wand of the Archlich","slot":"RANGED"}],"character":{"race":"Human","name":"Xmcdizzle","faction":"Alliance","level":80,"gameClass":"Mage"},"name":"Xmcdizzle QuickSimExport"}

**Comment by Jed-Giblin** [comment]
hey @xmcdizzle I pushed some missing enchants into the data file if you want to test on your character. You can manually update the file with a raw copy/paste or use git.

The file is here: https://github.com/Jed-Giblin/quicksim/blob/main/quicksim/data.lua

Ive decided not to support shoulder enchants, because the itemDB + enchantID + spellID is a mess for those items.

**Comment by xmcdizzle** [comment]
Word. I’ll give it a go when I get a chance. 

Thanks for the assistance!
